rest/util.go

```diff
@@ -35,16 +34,24 @@ func SendJSON(writer http.ResponseWriter, code int, value interface{}) error {
 	return encoder.Encode(value)
 }
 
+// NewJSONResponse turns plain object into a byte array representing JSON value and wraps it in filter.Response
+func NewJSONResponse(code int, value interface{}) (*filter.Response, error) {
+	headers := http.Header{}
+	headers.Add("Content-Type", "application/json")
+	body, err := json.Marshal(value)
```

Comment:
we should panic all internal errors during the requests and have a recovery filter which converts a panic to a 500 response (improvement in another PR)

Reply:
https://github.com/Peripli/service-manager/issues/67 - I've created an improvement for that sometime ago

Reply:
The result is the same - status 500